js_modules/dagster-ui/packages/ui-core/src/asset-data/AssetLiveDataProvider.tsx

```diff
@@ -136,9 +125,14 @@ export const AssetLiveDataProvider = ({children}: {children: React.ReactNode}) =
 
   useThrottledEffect(
     () => {
-      const assetKeyTokens = new Set(allObservedKeys.map(tokenForAssetKey));
-      const dataForObservedKeys = allObservedKeys
-        .map((key) => AssetBaseData.manager.getCacheEntry(tokenForAssetKey(key))!)
+      const assetKeyTokensArray = [
+        ...staleKeysObserved.current.flatMap((key) => Array.from(key)),
```

Comment:
Stupid nit but technically this is mapping over `keySet` containing keys, not `key` items

Reply:
ah yeah 😅